lib/react.dart

```diff
@@ -822,6 +822,56 @@ class SyntheticMouseEvent extends SyntheticEvent {
             type) {}
 }
 
+class SyntheticPointerEvent extends SyntheticEvent {
+  final num pointerId;
+  final num width;
+  final num height;
+  final num pressure;
+  final num tangentialPressure;
+  final num tiltX;
+  final num tiltY;
+  final num twist;
+  final String pointerType;
+  final bool isPrimary;
+
+  SyntheticPointerEvent(
+      bubbles,
```

Comment:
These arguments without the `this.` need typing, because unfortunately they don't get their types inferred 😿